src/devices/Common/Iot/Device/Graphics/BitmapImage.cs

```diff
@@ -7,8 +7,18 @@
 
 namespace Iot.Device.Graphics
 {
+    /// <summary>
+    /// Represents bitmap image
+    /// </summary>
     public abstract class BitmapImage
     {
+        /// <summary>
+        /// Constructs BitmapImage instance
```

Comment:
Constructors normally use a sentence like this:

```suggestion
        /// Initializes a <see cref="T:Iot.Device.Graphics.BitmapImage" /> instance with the specified data, width, height and stride.
```